src/redturtle/prenotazioni/content/prenotazioni_folder.py

```diff
@@ -593,7 +613,24 @@ def data_validation(data):
         defaultFactory=notify_on_refuse_message_default_factory,
         required=False,
     )
-
+    notify_as_reminder_subject = schema.TextLine(
+        title=_(
+            "notify_as_reminder_subject",
+            default="Booking reminder subject.",
+        ),
+        description=_("notify_as_reminder_subject_help", default=""),
```

Comment:
Add a description about what this field do.

Reply:
The field name is aligned with other fields name style from the same tab.

Reply:
Also due to volto rendering problems we do not use the help text in this tab

Reply:
wat?? btw add a description anyway, we could not use volto always

Reply:
This version of package is supposed to be used with volto frontend and it does not work well on the plone site.

Reply:
Solved

Reply:
it works on Plone and if Volto has some problems showing descriptions it's not our problem